do-netlify-stats-aggr.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+git checkout netlify-stats && git pull --ff-only
+git checkout main && git pull --ff-only
+
+bundle exec --gemfile collect-stats/Gemfile collect-stats/collect-stats.rb
```

Comment:
If you're testing and this hasn't been merged yet, you'll want to run this `bundle exec` command instead of the script (as it won't be present in the main branch yet)